### PR TITLE
add doc for schema cache

### PR DESCRIPTION
**Pull request by wjhuang2016**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs-cn/pull/18716

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: schema-cache.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: schema 缓存
+aliases: ['/docs-cn/dev/information-schema-cache']
+summary: TiDB 对于 schema 信息采用缓存机制，在大量数据库和表的场景下能够显著减少 schema 信息的内存占用以及提高性能。
+---
+
+# schema 缓存
+
+在一些多租户的场景下，可能会存在几十万甚至上百万个数据库和表。这些数据库和表的 schema 信息如果全部加载到内存中，一方面会占用大量的内存，另一方面会导致相关的访问性能变差。
+为了解决这个问题，TiDB 引入了 schema 缓存机制。采用类似于 LRU 的机制，只将最近用到的数据库和表的 schema 信息中缓存到内存中。
```
Comment (fzzf678) [inline schema-cache.md:10]:
```suggestion
为了解决这个问题，TiDB 引入了 schema 缓存机制。采用类似于 LRU 的机制，只将最近用到的数据库和表的 schema 信息缓存到内存中。
```

File: schema-cache.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## 配置
+
+可以通过配置系统变量 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入) 来打开 schema 缓存特性。
```
Comment (Benjamin2037) [inline schema-cache.md:14]: 是不是也给一下监控界面和指标的说明？

File: schema-cache.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+## 最佳实践
+
+- 在大量数据库和表的场景下，建议打开 schema 缓存特性。
+- 建议关闭 [`performance.force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v657-和-v710-版本开始引入) 以减少 TiDB 的启动时间。
+- 建议关闭 [`split-table`](/tidb-configuration-file.md#split-table) 以减少 region 数量，从而降低 TiKV 的内存。
+
+## 已知限制
+
+- 在大量数据库和表的场景下，统计信息不一定能够及时收集。
+- 在大量数据库和表的场景下，一些元数据信息的访问会变慢。
```
Comment (tangenta) [inline schema-cache.md:25]: TiDB 启动速度变慢要提吗

File: schema-cache.md (hunk 2 above)
Comment (Frank945946) [reply schema-cache.md:14]: +1

File: schema-cache.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+在一些多租户的场景下，可能会存在几十万甚至上百万个数据库和表。这些数据库和表的 schema 信息如果全部加载到内存中，一方面会占用大量的内存，另一方面会导致相关的访问性能变差。
+为了解决这个问题，TiDB 引入了 schema 缓存机制。采用类似于 LRU 的机制，只将最近用到的数据库和表的 schema 信息中缓存到内存中。
+
+## 配置
+
+可以通过配置系统变量 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入) 来打开 schema 缓存特性。
+
+## 最佳实践
+
+- 在大量数据库和表的场景下，建议打开 schema 缓存特性。
```
Comment (Frank945946) [inline schema-cache.md:18]: 可能要给一个参考值，比如多大内存，多少库和表的场景下建议开启。

File: schema-cache.md (hunk 3 above)
Comment (Frank945946) [reply schema-cache.md:25]: 还有来回切换 v2 和 v1 ，建议把尽可能的限制都写上，后面逐步解掉。

File: schema-cache.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- 建议关闭 [`performance.force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v657-和-v710-版本开始引入) 以减少 TiDB 的启动时间。
+- 建议关闭 [`split-table`](/tidb-configuration-file.md#split-table) 以减少 region 数量，从而降低 TiKV 的内存。
+
+## 已知限制
+
+- 在大量数据库和表的场景下，统计信息不一定能够及时收集。
+- 在大量数据库和表的场景下，一些元数据信息的访问会变慢。
+- 在大量数据库和表的场景下，TiDB 的启动时间会变长。开启 schema 缓冲能够缓解这个问题。
+- 在大量数据库和表的场景下，切换 schema 缓存开关需要等待一段时间。
```
Comment (GMHDBJD) [inline schema-cache.md:26]: - 在大量数据库和表的场景下，全量列举所有元数据信息的相关操作会变慢，如：
  - `SHOW FULL TABLES`
  - `FLASHBACK`
  - `ALTER TABLE ... SET TIFLASH MODE ...`

File: schema-cache.md (hunk 2 above)
Comment (tiancaiamao) [reply schema-cache.md:13]: +1

File: schema-cache.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+- 在大量数据库和表的场景下，全量列举所有元数据信息的相关操作会变慢，如：
+    - `SHOW FULL TABLES`
+    - `FLASHBACK`
+    - `ALTER TABLE ... SET TIFLASH MODE ...`
```
Comment (CbcWestwolf) [inline schema-cache.md:32]:
```suggestion
在大量数据库和表的场景下，schema cache 有以下限制：
- 统计信息不一定能够及时收集。
- 一些元数据信息的访问会变慢。
- TiDB 的启动时间会变长。开启 schema 缓冲能够缓解这个问题。
- 切换 schema 缓存开关需要等待一段时间。
- 全量列举所有元数据信息的相关操作会变慢，如：
    - `SHOW FULL TABLES`
    - `FLASHBACK`
    - `ALTER TABLE ... SET TIFLASH MODE ...`
```

File: schema-cache.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Operation 来查看 schema 缓存的命中率。如果命中率较低，可以调大 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入)。
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Size 来查看当前使用的 schema 缓存的大小。
+- 建议关闭 [`performance.force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v657-和-v710-版本开始引入) 以减少 TiDB 的启动时间。
+- 建议关闭 [`split-table`](/tidb-configuration-file.md#split-table) 以减少 region 数量，从而降低 TiKV 的内存。
+
+## 已知限制
+
+- 在大量数据库和表的场景下，统计信息不一定能够及时收集。
+- 在大量数据库和表的场景下，一些元数据信息的访问会变慢。
+- 在大量数据库和表的场景下，TiDB 的启动时间会变长。开启 schema 缓冲能够缓解这个问题。
```
Comment (fzzf678) [inline schema-cache.md:27]:
```suggestion
- 在大量数据库和表的场景下，TiDB 的启动时间会变长。开启 schema 缓存能够缓解这个问题。
```

File: schema-cache.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+## 配置
+
+可以通过配置系统变量 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入) 来打开 schema 缓存特性。
+
+## 最佳实践
+
+- 在大量数据库和表的场景下（例如10万以上的数据库和表数量）或者当数据库和表的数量大到影响系统性能时，建议打开 schema 缓存特性。
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Operation 来查看 schema 缓存的命中率。如果命中率较低，可以调大 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入)。
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Size 来查看当前使用的 schema 缓存的大小。
```
Comment (fzzf678) [inline schema-cache.md:19]: 要不要贴个截图

Comment (wjhuang2016) [reply schema-cache.md:19]: 感觉不需要，用户如果熟悉监控的话也能找到

File: schema-cache.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+---
+title: schema 缓存
+aliases: ['/docs-cn/dev/information-schema-cache']
+summary: TiDB 对于 schema 信息采用缓存机制，在大量数据库和表的场景下能够显著减少 schema 信息的内存占用以及提高性能。
+---
+
+# schema 缓存
+
+在一些多租户的场景下，可能会存在几十万甚至上百万个数据库和表。这些数据库和表的 schema 信息如果全部加载到内存中，一方面会占用大量的内存，另一方面会导致相关的访问性能变差。为了解决这个问题，TiDB 引入了 schema 缓存机制。采用类似于 LRU 的机制，只将最近用到的数据库和表的 schema 信息缓存到内存中。
+
```
Comment (hfxsd) [inline schema-cache.md:10]:
```suggestion

> **警告：**
>
> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。

```

File: schema-cache.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+## 配置
+
+可以通过配置系统变量 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入) 来打开 schema 缓存特性。
+
+## 最佳实践
+
+- 在大量数据库和表的场景下（例如10万以上的数据库和表数量）或者当数据库和表的数量大到影响系统性能时，建议打开 schema 缓存特性。
```
Comment (hfxsd) [inline schema-cache.md:17]:
```suggestion
- 在大量数据库和表的场景下（例如 10 万以上的数据库和表数量）或者当数据库和表的数量大到影响系统性能时，建议打开 schema 缓存特性。
```

File: schema-cache.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+## 已知限制
+
+- 在大量数据库和表的场景下，统计信息不一定能够及时收集。
+- 在大量数据库和表的场景下，一些元数据信息的访问会变慢。
+- 在大量数据库和表的场景下，TiDB 的启动时间会变长。开启 schema 缓存能够缓解这个问题。
+- 在大量数据库和表的场景下，切换 schema 缓存开关需要等待一段时间。
+- 在大量数据库和表的场景下，全量列举所有元数据信息的相关操作会变慢，如：
+    - `SHOW FULL TABLES`
+    - `FLASHBACK`
+    - `ALTER TABLE ... SET TIFLASH MODE ...`
```
Comment (hfxsd) [inline schema-cache.md:32]:
```suggestion
在大量数据库和表的场景下，有以下已知问题：

- 统计信息不一定能够及时收集。
- 一些元数据信息的访问会变慢。
- TiDB 的启动时间会变长。开启 schema 缓存能够缓解该问题。
- 切换 schema 缓存开关需要等待一段时间。
- 全量列举所有元数据信息的相关操作会变慢，如：

    - `SHOW FULL TABLES`
    - `FLASHBACK`
    - `ALTER TABLE ... SET TIFLASH MODE ...`
```

File: schema-cache.md [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+可以通过配置系统变量 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入) 来打开 schema 缓存特性。
+
+## 最佳实践
+
+- 在大量数据库和表的场景下（例如 10 万以上的数据库和表数量）或者当数据库和表的数量大到影响系统性能时，建议打开 schema 缓存特性。
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Operation 来查看 schema 缓存的命中率。如果命中率较低，可以调大 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入)。
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Size 来查看当前使用的 schema 缓存的大小。
+- 建议关闭 [`performance.force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v657-和-v710-版本开始引入) 以减少 TiDB 的启动时间。
+- 建议关闭 [`split-table`](/tidb-configuration-file.md#split-table) 以减少 region 数量，从而降低 TiKV 的内存。
```
Comment (Frank945946) [inline schema-cache.md:25]: 建议加上场景“如果需要创建大量的表（例如 10 万张以上），建议将此参数设置为 false”

File: schema-cache.md [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+
+## 已知限制
+
+在大量数据库和表的场景下，有以下已知问题：
```
Comment (Frank945946) [inline schema-cache.md:29]: 这个是开启 schema cache v2 后在大量数据库和表的场景下的限制么？看的像，但是又不像，比如这一条“TiDB 的启动时间会变长。开启 schema 缓存能够缓解该问题。”

Comment (wjhuang2016) [reply schema-cache.md:29]: 这个算是大量库表下的问题，也不完全是限制

File: schema-cache.md (hunk 12 above)
Comment (wjhuang2016) [reply schema-cache.md:25]: 你用建议模式改下？

File: schema-cache.md [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+- 在大量数据库和表的场景下（例如 10 万以上的数据库和表数量）或者当数据库和表的数量大到影响系统性能时，建议打开 schema 缓存特性。
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Operation 来查看 schema 缓存的命中率。如果命中率较低，可以调大 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入)。
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Size 来查看当前使用的 schema 缓存的大小。
+- 建议关闭 [`performance.force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v657-和-v710-版本开始引入) 以减少 TiDB 的启动时间。
+- 如果需要创建大量的表（例如 10 万张以上），建议将此参数设置为 false [`split-table`](/tidb-configuration-file.md#split-table) 以减少 region 数量，从而降低 TiKV 的内存。
+
+## 已知限制
+
+在大量数据库和表的场景下，有以下已知问题：
+
```
Comment (Frank945946) [inline schema-cache.md:30]:
```suggestion
- 当需要被访问的表是没有规律的，如 t1 访问一批表，t2 访问另外一批表，而且设置的 tidb_schema cache size 较小时，会导这些 schema 信息被频繁地被逐出，频繁地被缓存，造成性能抖动。该特性比较适合被频繁访问的库和表是相对固定的场景。
```

File: schema-cache.md [hunk 15]
```diff
@@ -0,0 +1,38 @@
+---
+title: schema 缓存
+aliases: ['/docs-cn/dev/information-schema-cache']
+summary: TiDB 对于 schema 信息采用基于 LRU 的缓存机制，在大量数据库和表的场景下能够显著减少 schema 信息的内存占用以及提高性能。
+---
+
+# schema 缓存
+
+在一些多租户的场景下，可能会存在几十万甚至上百万个数据库和表。这些数据库和表的 schema 信息如果全部加载到内存中，一方面会占用大量的内存，另一方面会导致相关的访问性能变差。为了解决这个问题，TiDB 引入了类似于 LRU 的机制 schema 缓存机制。只将最近用到的数据库和表的 schema 信息缓存到内存中。
```
Comment (Frank945946) [inline schema-cache.md:9]:
```suggestion
在一些多租户的场景下，可能会存在几十万甚至上百万个数据库和表。这些数据库和表的 schema 信息如果全部加载到内存中，一方面会占用大量的内存，另一方面会导致相关的访问性能变差。为了解决这个问题，TiDB 引入了类似于 LRU 的 schema 缓存机制。只将最近用到的数据库和表的 schema 信息缓存到内存中。
```

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: schema-cache.md [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+
+可以通过配置系统变量 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入) 来打开 schema 缓存特性。
+
+## 最佳实践
+
+- 在大量数据库和表的场景下（例如 10 万以上的数据库和表数量）或者当数据库和表的数量大到影响系统性能时，建议打开 schema 缓存特性。
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Operation 来查看 schema 缓存的命中率。如果命中率较低，可以调大 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入)。
+- 可以通过观测 TiDB 监控中 Schema load 下的子面板 Infoschema v2 Cache Size 来查看当前使用的 schema 缓存的大小。
+- 建议关闭 [`performance.force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v657-和-v710-版本开始引入) 以减少 TiDB 的启动时间。
+- 如果需要创建大量的表（例如 10 万张以上），建议将此参数设置为 false [`split-table`](/tidb-configuration-file.md#split-table) 以减少 region 数量，从而降低 TiKV 的内存。
```
Comment (hfxsd) [inline schema-cache.md:25]:
```suggestion
- 如果需要创建大量的表（例如 10 万张以上），建议将参数 [`split-table`](/tidb-configuration-file.md#split-table) 设置为 `false` 以减少 Region 数量，从而降低 TiKV 的内存。
```

File: schema-cache.md [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+- 可以通过观测 TiDB 监控中 **Schema load** 下的子面板 **Infoschema v2 Cache Operation** 来查看 schema 缓存的命中率。如果命中率较低，可以调大 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入)。
+- 可以通过观测 TiDB 监控中 **Schema load** 下的子面板 **Infoschema v2 Cache Size** 来查看当前使用的 schema 缓存的大小。
+- 建议关闭 [`performance.force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v657-和-v710-版本开始引入) 以减少 TiDB 的启动时间。
+- 如果需要创建大量的表（例如 10 万张以上），建议将参数 [`split-table`](/tidb-configuration-file.md#split-table) 设置为 `false` 以减少 Region 数量，从而降低 TiKV 的内存。
+
+## 已知限制
+
+在大量数据库和表的场景下，有以下已知问题：
+
+- 当需要被访问的表是没有规律的，如 t1 访问一批表，t2 访问另外一批表，而且设置的 tidb_schema cache size 较小时，会导这些 schema 信息被频繁地被逐出，频繁地被缓存，造成性能抖动。该特性比较适合被频繁访问的库和表是相对固定的场景。
```
Comment (hfxsd) [inline schema-cache.md:31]:
```suggestion
- 当需要被访问的表是没有规律的，如 time1 访问一批表，time2 访问另外一批表，而且设置的 `tidb_schema cache size` 较小时，会导这些 schema 信息被频繁地被逐出，频繁地被缓存，造成性能抖动。该特性比较适合被频繁访问的库和表是相对固定的场景。
```

File: TOC.md [hunk 18]
```diff
@@ -1026,6 +1026,7 @@
         - [`schema_unused_indexes`](/sys-schema/sys-schema-unused-indexes.md)
     - [元数据锁](/metadata-lock.md)
     - [TiDB 加速建表](/accelerated-table-creation.md)
+    - [schema 缓存](/schema-cache.md)
```
Comment (lilin90) [inline TOC.md:1029]:
```suggestion
    - [Schema 缓存](/schema-cache.md)
```

File: schema-cache.md [hunk 19]
```diff
@@ -0,0 +1,39 @@
+---
+title: schema 缓存
```
Comment (lilin90) [inline schema-cache.md:2]:
```suggestion
title: Schema 缓存
```

File: schema-cache.md [hunk 20]
```diff
@@ -0,0 +1,39 @@
...
+aliases: ['/docs-cn/dev/information-schema-cache']
+summary: TiDB 对于 schema 信息采用基于 LRU 的缓存机制，在大量数据库和表的场景下能够显著减少 schema 信息的内存占用以及提高性能。
+---
+
+# schema 缓存
```
Comment (lilin90) [inline schema-cache.md:7]:
```suggestion
# Schema 缓存
```

File: schema-cache.md [hunk 21]
```diff
@@ -0,0 +1,39 @@
+---
+title: Schema 缓存
+aliases: ['/docs-cn/dev/information-schema-cache']
+summary: TiDB 对于 schema 信息采用基于 LRU 的缓存机制，在大量数据库和表的场景下能够显著减少 schema 信息的内存占用以及提高性能。
```
Comment (lilin90) [inline schema-cache.md:4]: 如果是第一次提供相关内容的新文档页面，不需要加 aliases。请确认下~
```suggestion
title: Schema 缓存
summary: TiDB 对于 schema 信息采用基于 LRU 的缓存机制，在大量数据库和表的场景下能够显著减少 schema 信息的内存占用以及提高性能。
```

File: schema-cache.md [hunk 22]
```diff
@@ -0,0 +1,38 @@
...
+
+# Schema 缓存
+
+在一些多租户的场景下，可能会存在几十万甚至上百万个数据库和表。这些数据库和表的 schema 信息如果全部加载到内存中，一方面会占用大量的内存，另一方面会导致相关的访问性能变差。为了解决这个问题，TiDB 引入了类似于 LRU 的 schema 缓存机制。只将最近用到的数据库和表的 schema 信息缓存到内存中。
+
+> **警告：**
+>
+> 该功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
+
+## 配置
```
Comment (lilin90) [inline schema-cache.md:14]:
```suggestion
## 配置 schema 缓存
```

File: schema-cache.md [hunk 23]
```diff
@@ -0,0 +1,38 @@
...
+- 可以通过观测 TiDB 监控中 **Schema load** 下的子面板 **Infoschema v2 Cache Operation** 来查看 schema 缓存的命中率。如果命中率较低，可以调大 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入)。
+- 可以通过观测 TiDB 监控中 **Schema load** 下的子面板 **Infoschema v2 Cache Size** 来查看当前使用的 schema 缓存的大小。
+- 建议关闭 [`performance.force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v657-和-v710-版本开始引入) 以减少 TiDB 的启动时间。
+- 如果需要创建大量的表（例如 10 万张以上），建议将参数 [`split-table`](/tidb-configuration-file.md#split-table) 设置为 `false` 以减少 Region 数量，从而降低 TiKV 的内存。
+
+## 已知限制
+
+在大量数据库和表的场景下，有以下已知问题：
+
+- 当需要被访问的表是没有规律的，如 time1 访问一批表，time2 访问另外一批表，而且设置的 `tidb_schema cache size` 较小时，会导这些 schema 信息被频繁地被逐出，频繁地被缓存，造成性能抖动。该特性比较适合被频繁访问的库和表是相对固定的场景。
```
Comment (lilin90) [inline schema-cache.md:30]:
```suggestion
- 当需要被访问的表是没有规律的，如 time1 访问一批表，time2 访问另外一批表，而且设置的 `tidb_schema_cache_size` 较小时，会导这些 schema 信息被频繁地被逐出，频繁地被缓存，造成性能抖动。该特性比较适合被频繁访问的库和表是相对固定的场景。
```

File: schema-cache.md [hunk 24]
```diff
@@ -0,0 +1,38 @@
...
+- 可以通过观测 TiDB 监控中 **Schema load** 下的子面板 **Infoschema v2 Cache Operation** 来查看 schema 缓存的命中率。如果命中率较低，可以调大 [`tidb_schema_cache_size`](/system-variables.md#tidb_schema_cache_size-从-v800-版本开始引入)。
+- 可以通过观测 TiDB 监控中 **Schema load** 下的子面板 **Infoschema v2 Cache Size** 来查看当前使用的 schema 缓存的大小。
+- 建议关闭 [`performance.force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v657-和-v710-版本开始引入) 以减少 TiDB 的启动时间。
+- 如果需要创建大量的表（例如 10 万张以上），建议将参数 [`split-table`](/tidb-configuration-file.md#split-table) 设置为 `false` 以减少 Region 数量，从而降低 TiKV 的内存。
+
+## 已知限制
+
+在大量数据库和表的场景下，有以下已知问题：
+
+- 当需要被访问的表是没有规律的，如 time1 访问一批表，time2 访问另外一批表，而且设置的 `tidb_schema_cache_size` 较小时，会导这些 schema 信息被频繁地被逐出，频繁地被缓存，造成性能抖动。该特性比较适合被频繁访问的库和表是相对固定的场景。
```
Comment (lilin90) [inline schema-cache.md:30]:
```suggestion
- 当对表的访问没有规律，如 time1 访问一批表，time2 访问另外一批表，而且设置的 `tidb_schema_cache_size` 较小时，会导这些 schema 信息被频繁地被逐出，频繁地被缓存，造成性能抖动。该特性比较适合被频繁访问的库和表是相对固定的场景。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**